sigopt/dict_run_context.py

```diff
@@ -0,0 +1,41 @@
+from .run_context import BaseRunContext
+import copy
+
+class DictRunContext(BaseRunContext):
```

Comment:
Rather call this `OfflineRunContext` or `LocalRunContext`

Reply:
Changed to 'LocalRunContext'